feat(serverless): edge faq (#4161)
scaleway/docs-content: faq/serverless-functions.mdx

Explore all Scaleway products in the console and select the right product for your use case.

Some products are not listed but for example, on specific use cases Secret Manager can help you to store informations that requires versioning.
</Message>

## Can I use Serverless Functions with Edge Services?

> For the moment there are no native integrations between both products.

Reviewer: ```suggestion
You cannot use Serverless Functions with Edge Services because there are no native integrations between the two products yet.
```

> ## Can I use IP address of Serverless Function?

Reviewer: ```suggestion
## Can I use the IP address of a Serverless Function?
```

> No, by design on Serverless compute ressources it's not possible to guarantee static IPs.

Reviewer: ```suggestion
By design, it is not possible to guarantee static IPs on Serverless compute resources.
```
